test/lit/passes/remove-unused-types-public-tags.wast

```diff
@@ -54,4 +54,31 @@
   (global $other-1 (ref null $other-1) (ref.null none))
   (global $other-2 (ref null $other-2) (ref.null none))
 )
+
+;; Neither imported nor exported tag. Now the tag type is private.
 ;; CHECK:      (export "" (tag $exported))
```

Comment:
Seems confusing to have "nor exported" right above a CHECK for an export. Maybe move the comment into the next module, or the CHECK can be moved?

Reply:
Ah, this is unfortunate ordering from the test update script. I'll reorder the test cases to fix it.